templates/docs/examples/patterns/image/container/highlighted.html

```diff
@@ -0,0 +1,10 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Image container / Highlighted{% endblock %}
+
+{% block standalone_css %}patterns_image{% endblock %}
+
+{% block content %}
+<div class="p-image-container is-highlighted u-align--center">
```

Comment:
Is the `u-align--center` needed? This should be built into the component ideally.

Reply:
Yes I built this into the common styling and forgot to remove it from the highlighted example as I did with the aspect ratio examples. Thanks; removed!